pkg/qust/patch_utils.go

```diff
@@ -110,7 +110,7 @@ func getSelectivePatchTemplate(name string) *config.SelectivePatch {
 		Metadata: &config.CustomMetadata{
 			Name: name,
 		},
-		Enabled: true,
+		Enabled: false,
```

Comment:
we cant change this value as this method used by multiple functions.
better to create another method with enabled: false

when we have `enabled: false` it is missing while write into disk, so we may need `enabled: "false"` something like this.